services/api-server/src/simcore_service_api_server/api/routes/solvers.py

```diff
@@ -265,7 +265,8 @@ async def list_solver_ports(
 @router.get(
     "/{solver_key:path}/releases/{version}/pricing_plan",
     response_model=ServicePricingPlanGet,
-    include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
+    description="Gets solver pricing plan\n\n"
+    + FMSG_CHANGELOG_NEW_IN_VERSION.format("0.7"),
```

Comment:
Maybe this can be read from the `VERSION` file in the meta module. This way you do not forge to bump it and is automatically updated?

Reply:
@GitHK the version in the metadata changes and this has to stay. You can see this as a "bump" of the current version.